### PR TITLE
feat(setup): update inputs and align form-usage

### LINKED ISSUE #703: Setup Redesign: InputPassword.tsx
grafik

grafik

grafik

grafik

### PR DESCRIPTION
**Pull request by escapedcat**
Fixes #703

draft:
image

design:
image

- what is the empty icon dot above?
- width/padding/margin of all forms should to be adjusted later
- color of all inputs should to be adjusted later

### CODE REVIEW

File: src/i18n/langs/en.json [hunk 1]
```diff
@@ -277,15 +277,15 @@
       "no_and_shutdown": "No and Shutdown",
       "ok": "OK",
       "other_options": "Other Options",
-      "password_a_details": "Use this password for login to your RaspiBlitz.",
+      "password_a_details": "Also known as Password A. Use this password for login to your RaspiBlitz.",
```
Comment (cstenglein) [inline src/i18n/langs/en.json:280]: I played around with that so we can add the coloring like the mockups.

See https://react.i18next.com/latest/trans-component#samples

**Review by cstenglein — APPROVED** [review]

File: src/i18n/langs/en.json (hunk 1 above)
Comment (escapedcat) [reply src/i18n/langs/en.json:280]: Yeah, about that...
I think it's a bit too close to a link tbh.

Link:
image

Strong:
image

wdyt? Is this an issue?

Comment (cstenglein) [reply src/i18n/langs/en.json]: Would rather change the link color as you did yes 

The other passwords have other colors so it will differ from a link

Comment (escapedcat) [reply src/i18n/langs/en.json]: Wait, I thought those were color variants but we decided to go with blue :D So each color is different? Ahhhh, ok.

Should we underline "real" links? I like that. Makes it obvious.

Comment (cstenglein) [reply src/i18n/langs/en.json]: Yes, the color variants are different so it makes a difference for the user. :)

Yes, links should be unterlined IMO: https://nextui.org/docs/components/link#underline

Maybe we can put that into the config or something to it always applies?

Comment (escapedcat) [reply src/i18n/langs/en.json]: done

**Review by cstenglein — APPROVED** [review]

**Pull request merged**